neuro_extras/main.py

```diff
@@ -458,7 +413,7 @@ async def _nonstorage_cp(
     elif source_url.scheme == "" and destination_url.scheme == "":
         command = "rclone"
         args = [
-            "copy",  # TODO: investigate usage of 'sync' for potential speedup.
+            "copyto",  # TODO: investigate usage of 'sync' for potential speedup.
```

Comment:
is this a typo ?

Reply:
No, see brief diff [explanation](https://www.reddit.com/r/seedboxes/comments/6i1ahb/rclone_difference_between_copy_and_copyto/).

Reply:
ok so this is needed by the logic of this PR. thanks